### DISCUSSION TITLE
Failed to match keys when loading Lora?

### DISCUSSION THREAD

**Discussion by mark-wd** [vladmandic/sdnext#1104] [General]
What should I understand from this error that spews out several A4 pages worth of error text but doesn't actually crash any aspect of webui?

**Comment by vladmandic** [comment]
i'll change print so it prints number of mismatches, not each mismatch as getting several pages is not useful.
but in general, each entry is part of lora checkpoint loaded from disk that is not understood by lora module - and those parts are basically printed as warning and then ignored. 

which means you can have lora trained with 100s of different layers, but only 50% (or 90% or 10%) of them actually used.

what are all other not-understood layers?
99% chance, what you're loading is not a basic lora. try loading is as `<lyco:name>`.
for a little bit more details, read <https://github.com/kohya-ss/sd-scripts/issues/397#issuecomment-1555020023>

**Comment by mark-wd** [comment]
Thanks for the great explanation.
